src/components/organisms/NewFeed/index.js

```diff
@@ -0,0 +1,213 @@
+import React, { useEffect, useState } from "react";
+import PropTypes from "prop-types";
+
+import { useSelector, useDispatch } from "react-redux";
+import { useHistory } from "react-router-dom";
+import styled from "styled-components";
+
+import GooglePlacesAutocomplete, { geocodeByAddress, getLatLng } from "react-google-places-autocomplete";
+
+import { getAddressFromLatLng, addNewFeed } from "../../../api";
+import { Icon, Textarea } from "../../atoms";
+import { NewFeedHeader } from "../../molecules";
+import themes from "../../../theme/theme";
+import ImageUpload from "../ImageUpload";
+import { isTokenExpired } from "../../../utils";
+import { userSliceActions } from "../../../modules/slices/userSlice";
+
+const { colors } = themes;
+
+const Container = styled.div`
+  display: flex;
+  width: 90rem;
+  height: 60rem;
+  background: ${({ theme }) => theme.colors.white};
+`;
+
+const ImageWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 50rem;
+  height: 100%;
+  background: ${({ theme }) => theme.opacityColor.gray};
+`;
+
+const ContentsWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 40rem;
+  height: 100%;
+  padding: 1rem;
+`;
+
+const CloseWrapper = styled.div`
+  width: 100%;
+  text-align: right;
+  margin-bottom: 2rem;
+`;
+
+const HeaderWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  width: 100%;
+  margin-bottom: 1rem;
+  padding-left: 1rem;
+  padding-right: 2rem;
+`;
+
+const AddressWrapper = styled.div`
+  display: flex;
+  width: 100%;
+  margin-top: 2rem;
+  text-align: right;
+`;
+
+function NewFeed({ handleClose }) {
+  const [pictureUrl, setPictureUrl] = useState([]);
+  const [content, setContent] = useState("");
+  const [location, setLocation] = useState([]);
+  const [inputAddress, setInputAddress] = useState("");
+  const [photoAddress, setPhotoAddress] = useState("");
+  const [address, setAddress] = useState("Please enter your address");
+  const dispatch = useDispatch();
+  const history = useHistory();
+  const userInfo = useSelector((state) => state.user.data);
+
+  const getImage = (data) => {
+    const { imageUrl, locationFromMeta } = data;
+    setPictureUrl(imageUrl);
+    setLocation(locationFromMeta);
+  };
+
+  const handleTextChange = (e) => {
+    setContent(e.target.value);
+  };
+
+  useEffect(() => {
+    const getAddress = async () => {
+      try {
+        const addressFromPhoto = await getAddressFromLatLng(location);
+        setPhotoAddress(addressFromPhoto.slice(5));
+      } catch (err) {
+        console.error(err);
+      }
+    };
+
+    if (pictureUrl) {
+      getAddress();
+    }
+  }, [pictureUrl]);
+
+  useEffect(() => {
+    const getCoordinates = async () => {
+      try {
+        const response = await geocodeByAddress(String(inputAddress));
+        const result = await getLatLng(response[0]);
+        setLocation(result);
+        return result;
```

Comment:
```suggestion
        const result = await getLatLng(response[0]);
        
        setLocation(result);
        
        return result;
```
개행이 있으면 가독성에 더 좋을 것 같습니다.🙇🏻

Reply:
넵 의견 주신 내용으로 수정하겠습니다!